language/expressions.md

```diff
@@ -1840,4 +1840,103 @@ Example:
 
 The Selector Expression supports unfold/splat the same way as in Case Expression.
 
+Definition Expressions
+---
+The Puppet Language has several definition expressions:
+
+* Function definition
+* User defied resource dfinition
+* Host Class definition
+* Type Alias defiition
+
+The user defied resource definition and host class definition expressions are specified
+in [Catalog Expressions][2], and function definition in [Puppet Functions][3], and [Function API][4].
+
+### Type Alias Expression
+
+The Type Alias Expression makes it possible to assign a type definition to a type
+name, and use the new type name as a 100% equivalence to the assigned type.
+
+In this version of the specification, only the case of the initial letter in each name segment
+is significant; MyType is equivalent to MYTYPE, MytYPe, etc. This may change in a later release.
+
+The grammar is:
+
+```
+TypeAliasExpression
+  : 'type' QualifiedReference '=' Expression<Type>
+  ;
+```
+
+Example use:
+
+~~~
+type PositiveInts = Array[Integer[0, default]]
+$a = [1,2,3] ~= PositiveInts
+$b = Array[Integer[0, default]] == PositiveInts
+~~~
+
+Would set both `$a` and `$b` to `true`.
+
+Type references are autoloaded from the environment and modules. The autoloading rules are:
+
+* The name of the file must be in lower case.
+* No underscore `_` should be use to separate words in a camel cased name; the file for
```

Comment:
Thanks! Now Fixed on master branch
(tip, it s dead easy to use edit at github to create a PR for typo fixes like this).